pinc/comment_inclusions.inc

```diff
@@ -41,7 +44,14 @@ function parse_project_comments($comments) {
 			$i++;
 		}
 	}
-	return $comments;
+
+	if ($project->comment_format == 'html') {
```

Comment:
OH, this file uses tabs and not spaces. Yes, being consistent here is the way to go and I'll follow up with a clean-up commit for this and any other tab-based files later. I thought we'd gotten all of them but clearly not. Sorry for the confusion earlier.